.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
aren't spaces the standard (It's also easier to just make your tab button 2 spaces)?

```
indent_style = space

indent_size = 4

```
cheers!